### DISCUSSION TITLE
To check the code Faction(_side)

### DISCUSSION THREAD

**Discussion by pomikrus** [official-antistasi-community/A3-Antistasi#3453] [General]
Faction(_side) - Does it work for you?

https://community.bistudio.com/wiki/faction

Syntax:
faction unitName
Parameters:
unitName: 
Return Value:
String

**Comment by rautamiekka** [comment]
I guess it didn't occur to you there must be a reason for its very widespread usage:
https://github.com/official-antistasi-community/A3-Antistasi/blob/unstable/A3A/addons/core/Includes/common.inc#L9
